config/config_test.go

```diff
@@ -35,3 +42,56 @@ func (s *configSuite) TestPrefixed(c *C) {
 	c.Assert(s.tlc.policy("quux"), Equals, s.tlc.prefixed(rootPolicy, "quux"))
 	c.Assert(s.tlc.volume("foo", "bar", "quux"), Equals, s.tlc.prefixed(rootVolume, "foo", "bar", "quux"))
 }
+
+func (s *configSuite) TestDumpTarball(c *C) {
+	key := "/volplugin/foo"
+	value := "baz"
+
+	exec.Command("/bin/sh", "-c", "etcdctl set "+key+" "+value).Run()
```

Comment:
I understand but as we discussed elsewhere I suggested that this doesn't need to be changed because it's only used in one place. gocheck will output the code that errors so we should have a good idea without a new function to determine that it's this code.

FWIW, I am going to suggest we not implement this for now.